processing/src/main/java/io/druid/segment/serde/FloatGenericColumnPartSerde.java

```diff
@@ -65,15 +65,15 @@ public static SerializerBuilder serializerBuilder()
   public static class SerializerBuilder
   {
     private ByteOrder byteOrder = null;
-    private FloatColumnSerializer delegate = null;
+    private Serializer delegate = null;
```

Comment:
IndexMergerV9.createFloatColumnPartSerde passes a GenericColumnSerializer instead of FloatColumnSerializer, this is done in order to not have duplicate method to create column part serdes in IndexMergerV9 and Float/Long/DoubleMergerV9 classed.